03-Mains/Heaving_Water_mass_transformation_decomposition.ipynb

```diff
@@ -0,0 +1,2108 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    p1 = (temp_SO_pert-temp_SO_ctrl).plot(vmin=-0.5,vmax=0.5, cmap = cm.cm.balance,add_colorbar = False)
</pre><p><code>p1</code> gets overwritten by each plot. If you only need to save it for one of the subplots, it can be removed from the others. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>